### ISSUE TITLE
Rule Complaint

### ISSUE THREAD

**Issue by FurdenFuhrer** [EarthMC/Issue-Tracker#356]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when mods are being retarded about naming rules, sometimes enforce them and sometimes they don't. I also am frustrated with the whole nation range BS. This reset could have been great, but no, the mods have to ruin it by adding these dumb ass rules.
**Describe the solution you'd like**
Clear concise description? Actually listen to the community have heard very little support in favour of keeping these rules and a shit ton on removing them for good.
**Additional context**

**Comment by JuliusCaesar12** [comment]
I know, you can't have fictional nation names!? Then what's the point in it being a geopolitical sandbox?
And with the nation range, you can't conquer other lands without making them a vassal!

**Comment by Kyoris** [comment]
tbh i just don't like the nation naming rule - i mean why is _Baltoslavia_ not a legitimate name? like it's a region that perfectly describes our nation's location, and it is mentioned in the rules that _region names can be used_

**Comment by ghost** [comment]
The rules aren't that strict, but they have to be somewhat realistic. Yes, it is extremely sad that we can't have fictional names from videogames or other things, but you're just gonna have to get along with it. You can work around this problem by making a nation at the very edge of the map or make a nation with an area that isn't so popular.

The range isn't BS and it's PERFECTLY fair. I don't get why you think it's unfair. Let me give you an example. Imagine being the nation California and owning a town on the edge of the East coast. It's stupid and unrealistic. If you don't know the range it's 5000 blocks. Use an image editor, memorize points on the map (-5000x 5000x -5000y 5000y) and put points on them in the image editor. Make a transparent circle going over those points. Also the points are away from the capital, so wherever the capital the points move with it. This is how I found the nation range for my past nation California.

Also all the mod's rules aren't dumb and without them things would be way worse. The only rules that suck are the ones that censor language. Everything else is good so you can have a better experience.

Shut the fuck up and stop whining for nothing, please.

**Comment by Kyoris** [comment]
ehh the radius should increase with nation size and the nation naming rules are a bit too strict tbh

**Comment by NigelFarage666** [comment]
You are right nation range sucks ass. It ruined tons of great EMC Classic communites and limited many nations.

**Comment by JuliusCaesar12** [comment]
The U.S. can't take up its own territory because of nation range.

**Comment by FurdenFuhrer** [comment]
One of the main reasons I am upset with the nation range is that Fix announced it on the day of release. He said that the nation range would be able to take up all of North America. Guess what it doesn't.

**Comment by FurdenFuhrer** [comment]
He lied to his following and did all of these strict naming rules when no one was asking for em.

**Comment by FurdenFuhrer** [comment]
Also @laidang08 
Have you seen Europe. The nation range has not helped border gore what so ever.
